### DISCUSSION TITLE
part-ii/2017-5/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#387] [Question comments]
Part II 2017 Algebraic Topology: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2017-5/

**Comment by artie2000** [comment]
<details>
<summary>(b)</summary>
SvK gives

$G=\langle a,b\mid a^2\rangle$

Then use the universal properties of free groups and of group presentations to construct the quotient $F_2\to S_3$, then descend it to the map $G\to S_3$ they want.

</details>
<details>
<summary>(c) (i)</summary>
With notation as before,

$\ker\phi=\langle\langle b^3,bab^{-1}a\rangle\rangle\leq G$.

Consider $Y$ as drawn in the image below:
![algtop](https://user-images.githubusercontent.com/17950993/171519692-c73c68d4-5c0a-4c5e-a266-499490869fde.png)
Since the generators of $\ker\phi$ are loops from any basepoint, $\ker\phi\leq\pi(Y)$. On the other hand, $Y$ is a degree 6 cover, so $\ker\phi=\pi(Y)$ by Langrange's theorem. By the Galois correspondence, or GLP, $Y$ and $\hat X$ are covering-isomorphic.

</details>
<details>
<summary>(c) (ii),(iii)</summary>
(ii): Yes: retract each sphere to one of the paths on it. More precisely, you can map

$( \theta,\phi)\to( \theta,0)$
on each sphere (in spherical polars).

(iii): No. The fundamental group of a graph is free: contract its minimal spanning tree to get a quotient homotopy equivalence with a wedge of circles. However, the fundamental group of $\hat{X}$ is $S_3$.

</details>
